### PR TITLE
Add Teseve.app v0.4.0

### PR DESCRIPTION
**Pull request by leny**
Adding Teseve.app from [http://teseve.github.io](http://teseve.github.io).

I've followed all the guides, hope all is okay. Feel free to request any modification if needed.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 293a4f82bcbe1a35650335f55ce835c3b05cc580. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by leny** [comment]
Hi, thanks. I'm curious about the changes you made (specially to don't do errors next time). Can you explain the errors returned by Travis & how to fix them?

Thanks in advance,

**Comment by vitorgalvao** [comment]
All I did were additions and small tweaks (i.e. your contribution was fine and would’ve worked as was).

Travis errors was just Travis acting up (restarting it would likely make it pass).

**Comment by vitorgalvao** [comment]
I’ll anotate your cask in this PR, so you get a better idea of what was changed.

File: Casks/teseve.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'teseve' do
+  version '0.4.0'
+  sha256 'd5f1713b6f3155b46b4552bcfd777175b293990c0280bee1ebf4675be423fa3b'
+
+  url 'https://github.com/teseve/teseve/releases/download/0.4.0/Teseve-0.4.0-mac-x64.zip'
```
Comment (vitorgalvao) [inline Casks/teseve.rb]: Added a `# github.com/teseve/teseve was verified as official when first introduced to the cask` comment above this line ([another example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (vitorgalvao) [inline Casks/teseve.rb]: Used string interpolation here, so we just need to update `version` at the top, in the future.

Comment (vitorgalvao) [inline Casks/teseve.rb]: Also added [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by leny** [comment]
Ok, thanks for all the infos. :+1: 

Have a great week-end!

**Comment by vitorgalvao** [comment]
Thank you. You too.

**Pull request closed without merging**